src/main/java/org/fusesource/jansi/io/AnsiToHtmlProcessor.java

```diff
@@ -0,0 +1,68 @@
+package org.fusesource.jansi.io;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class AnsiToHtmlProcessor extends AnsiProcessor {
+    private boolean concealOn = false;
+    private HtmlAnsiOutputStream haos;
+
+    AnsiToHtmlProcessor(OutputStream os) {
+        super(os);
+    }
+
+    public HtmlAnsiOutputStream getHtmlAnsiOutputStream() {
+        return haos;
+    }
+
+    public void setHtmlAnsiOutputStream(HtmlAnsiOutputStream haos) {
+        this.haos = haos;
+    }
```

Comment:
The getter and setter should be removed.

Reply:
Cannot be removed